### PR TITLE
Breaking: Update dependencies, refactor TypeScript definition to CommonJS compatible export

### PR DESCRIPTION
**Pull request by BendingBender**
**Breaking change** for the same reasons as `PCancelable`.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -207,4 +207,3 @@ class Conf {
 }
 
 module.exports = Conf;
-module.exports.default = Conf;
```
Comment (sindresorhus) [inline index.js]: Is this a breaking change? I'm asking since this is being removed, but the PR title doesn't make it clear.

Comment (BendingBender) [reply index.js]: Sorry, I've written this in the description, not in the title. Will change.

Comment (sindresorhus) [reply index.js]: Oops, I missed that.

**Comment by sindresorhus** [comment]
I will not do a new release here yet. I have some other changes planned before the next major bump.

**Pull request merged**